src/app/zeko/da-layer/da_utils/src/lib.rs

```diff
@@ -0,0 +1,395 @@
+mod da_layer;
```

Comment:
This whole file could basically go away if you used JSON

Reply:
yeah #178